m3_json_schema.json

```diff
@@ -270,22 +270,13 @@
           },
           "indexing": {
             "type": "array",
-            "description": "a list of dynamic field names, taken from a controlled list; commonly used values are _tesim (stored searchable text), _teim (searchable text), _sim (string facetable), _ssm (string displayable)",
+            "description": "a list of Solr dynamic field names following the pattern property_name + suffix",
```

Comment:
TODO - make this backwards compatible. It should support both for now

Reply:
It doesn't make sense to do this because then it would be invalid for hyrax/hyku